sdk/src/main/java/com/stytch/sdk/StytchClient.kt

```
@@ -261,32 +262,32 @@ public object StytchClient {
      * @param sessionDurationMinutes - sessionDuration
      * @return AuthResponse from backend after calling any of the authentication methods
      */
-    public suspend fun handle(uri: Uri, sessionDurationMinutes: UInt): AuthResponse {
+    public suspend fun handle(uri: Uri, sessionDurationMinutes: UInt): DeeplinkHandledStatus {
         assertInitialized()
-        val result: AuthResponse
-        withContext(dispatchers.io) {
+        return withContext(dispatchers.io) {
             val token = uri.getQueryParameter(Constants.QUERY_TOKEN)
             if (token.isNullOrEmpty()) {
-                result = StytchResult.Error(StytchExceptions.Input("Magic link missing token"))
-                return@withContext
+                return@withContext DeeplinkHandledStatus.NotHandled(StytchErrorType.DEEPLINK_MISSING_TOKEN.message)
```

Wondering if the error is helpful here, or if we're just telling the developer that the deeplink should be handled elsewhere. I suppose they could ignore the error if they don't care. I'm mainly just wondering if we'd expect consumers of this to ever do anything with the error.

---

Yeah, I think for not handled, they'd probably just ignore it, but I think it might be good to explain _why_ it wasn't handled, in case they were expecting it to be.

---

sgtm